core/src/main/java/com/linecorp/armeria/common/stream/DefaultStreamMessage.java

```diff
@@ -168,8 +168,10 @@ protected void subscribe0(EventExecutor executor, SubscriptionOption[] options)
 
     /**
      * Invoked whenever a new demand is requested.
+     * @param newDemand Newly requested demand
+     * @param oldDemand Existing demand before adding newDemand
```

Comment:
Is there any reason that you add `oldDemand`? It doesn't seem like used. 🤔

Reply:
right, let me remove it.( I added it when using different way to implement this PR, but looks like I don't need it anymore and if some other implementation in the future need it, we can add it back)